actix-http/src/cookie/secure/private.rs

```diff
@@ -57,9 +60,14 @@ impl<'a> PrivateJar<'a> {
         let unsealed = open_in_place(&key, nonce, ad, 0, sealed)
             .map_err(|_| "invalid key/nonce/value: bad seal")?;
 
-        ::std::str::from_utf8(unsealed)
-            .map(|s| s.to_string())
-            .map_err(|_| "bad unsealed utf8")
+        if let Ok(unsealed_utf8) = str::from_utf8(unsealed) {
+            Ok(unsealed_utf8.to_string())
+        } else {
+            error!("Private cookie does not have utf8 content!
```

Comment:
Looks good, but I think it should be a `warn` macro rather than error.
It doesn't per se would indicate immediate error, but more or less a warning as original issue asks